opgincidentresponse/settings/base.py

```diff
@@ -4,6 +4,12 @@
 from django.core.exceptions import ImproperlyConfigured
 
 from response.slack.client import SlackClient
+from pathlib import Path
+from dotenv import load_dotenv
+
+current_path = Path(os.path.dirname(os.path.realpath(__file__)))
+env_path = current_path / "../../.env"
+load_dotenv(dotenv_path=env_path)
```

Comment:
This shouldn't be necessary if you have `.env` in the root directory

Reply:
Seeeee I was getting all sorts of issues running tests via docker so I reverted to using a virtualenv locally (.env at root in both instances).

Reply:
Ah, then you probably need to do something like `source .env` to get the environment variables into memory.

The `.env` is just a convenient way to store then, not something that should be loaded into the application.

Reply:
Ah, then you probably need to do something like `source .env` to get the environment variables into memory.

The `.env` is just a convenient way to store then, not something that should be loaded into the application.